roles/certificates/README.rst

```diff
@@ -0,0 +1,34 @@
+An ansible role to install and update CA certificates.
+
+**Role Variables**
+
+.. zuul:rolevar:: certificates_ca
+   :default: []
+   
+   This is a list wich contains the name and the certificate.
```

Comment:
s/wich/which

Also, what is the name needed for?

Reply:
Done